lib/Task.ex

```diff
@@ -1,24 +1,24 @@
 defmodule Mix.Tasks.PerformanceTest do
   use Mix.Task
 
-  @syntax_error_message "Wrong syntax! Try: mix performance_test --mode <push|pull|autodemand> --n <number of elements>
-   --howManyTries <how many tries> --tick <single try length [ms]> --initialGeneratorFrequency <frequency of the message generator in the first run>
-   --chosenMetrics <comma separated list of statistic names which should be saved> --reductions <number of reductions to be performed in each filter, while processing buffer>
-   OPTIONAL: [--shouldAdjustGeneratorFrequency, --shouldProducePlots, --shouldProvidemetricsHeader]
+  @syntax_error_message "Wrong syntax! Try: mix performance_test --mode <push|pull|autodemand> --numberOfElements <number of elements>
+   --howManyTries <how many tries> OPTIONAL: [--tick <single try length [ms], default: 10_000> --initialGeneratorFrequency <frequency of the message generator in the first run, default: 50_000 msg/s>
+   --chosenMetrics <comma separated list of statistic names which should be saved, default: ''> --reductions <number of reductions to be performed in each filter, while processing buffer, default: 10_000>
+   --shouldAdjustGeneratorFrequency, --shouldProducePlots, --shouldProvidemetricsHeader]
```

Comment:
I was unsure how to use these flags. Either lets add possible values set or maybe it is possible to pass a flag without arguments, it is sometimes done like that right. I mean, one has to pass `--shouldProducePlots` instead of `--shouldProducePlots true`
```suggestion
   --shouldAdjustGeneratorFrequency, --shouldProducePlots <true|false>, --shouldProvidemetricsHeader]
```

Reply:
I just give it a try and found out that this is a path to file where plots should be saved? Definitely, it was not clear to me :P

Reply:
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/27630111/157678112-0e32760d-71c6-43af-816e-13bcc7ddafb4.png">

Reply:
When it comes to --should* flags, they do not require `true`|`false` values - you can just pass `--shouldAdjustGeneratorFrequency` and it will work just as if you have passed `--shouldAdjustGeneratorFrequency true`

Reply:
the other problem you have mentioned is already solved by creating the missing directories :P